package.json

```diff
@@ -5,10 +5,11 @@
   "scripts": {
     "apps": "./script/app-list.sh",
     "analyze": "webpack-bundle-analyzer build/vagovprod/generated/stats.json",
-    "build": "node --max-old-space-size=4096 script/build.js",
+    "build": "npm run build:webpack && npm run build:content",
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype vagovprod --analyzer; npm run analyze",
+    "build:content": "node --max-old-space-size=4096 script/build-content.js",
     "build:content:test": "node script/test-cms-export.js",
-    "build-webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/standalone-webpack.config.js",
+    "build:webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/webpack.config.js --progress",
```

Comment:
Decided to rename this for consistency. Note that it's also now using the `webpack.config.js` instead of the former `standalone-webpack.config.js` now. This is because the `webpack.config.js` is _meant_ to be standalone now. (As opposed to being used by an instance of Webpack run through Metalsmith.)